src/ChainlinkOracle.sol

```diff
@@ -35,6 +35,12 @@ contract ChainlinkOracle is IOracle {
 
     /* CONSTRUCTOR */
 
+    /// @dev Here is the list of assumptions on the inputs that guarantees the oracle behaves as expected:
+    /// - Feeds are Chainlink-compliant or the address zero.
```

Comment:
```suggestion
    /// - Feeds are developed & maintained by Chainlink, or the address zero.
```

Reply:
Not really since we'll use redstone that is chainlink-compliant but not maintained and developed by them